CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Change Log
 
-3.9.0-rc1 (2017-12-26)
+3.9.0-rc2 (2018-01-04)
 ======================
 
 We are proud to announce the arrival of SuperCollider 3.9.0-rc1! Apologies
```

Comment:
should update here too. also are we adding to the changelog?

Reply:
ah, right.

there's a specific reason i'm not adding the two new entries -- the changelog is for changes against 3.8.1. since the bugs being fixed were only ever seen in the beta, we're basically acting as if they never happened.

the two bugs will appear in the release page and in the announcement to the list, though.

Reply:
got it! that makes sense